python/pyspark/ml/fpm.py

```diff
@@ -49,6 +49,32 @@ def getMinSupport(self):
         return self.getOrDefault(self.minSupport)
 
 
+class HasNumPartitions(Params):
+    """
+    Mixin for param support.
+    """
+
+    numPartitions = Param(
+        Params._dummy(),
+        "numPartitions",
+        """Number of partitions (at least 1) used by parallel FP-growth.
```

Comment:
Using ```"""``` to wrap doc here will get ```\n``` in generated Python API docs. You can update to use ```"``` referring to [discussion at here](https://github.com/apache/spark/pull/18089#discussion_r118280610).